### PR TITLE
removed mutex according to issue #7371 Revert b780da4

### PR DESCRIPTION
**Pull request by SyedSaifuddin045**

### CODE REVIEW

**Comment by stevecotton** [comment]
On the one hand, it's good to reformat the source code (and this does exactly match our clang-format style).

On the other hand, the reformatting changes `1 file changed, 6 insertions(+), 21 deletions(-)` into `1 file changed, 332 insertions(+), 370 deletions(-)`.

**Comment by CelticMinstrel** [comment]
I wouldn't be surprised if he just had the IDE set to format on save and didn't realize, or something.

**Comment by CelticMinstrel** [comment]
Also, maybe there's an error in the clang-format, because I thought our style guide was to not indent within namespaces? Although maybe anonymous namespaces are an exception…

**Comment by SyedSaifuddin045** [comment]
Ohh Really sorry,Yes I used the Linter for formatting the code.I will make sure it's off.

**Comment by CelticMinstrel** [comment]
If you could redo this without the linter that would be helpful.

Note that we don't have an issue with reformatting files (as long as it's to follow the style guide we use; there are probably a few clang-format changes required if using that). In future, if you want to use the linter, you can check in the reformatting in a separate commit before doing any actual work. The reason is that it makes it easier for us to review the change you made.

**Comment by SyedSaifuddin045** [comment]
I force pushed please check, unfortunately I couldn't get the previous formatting of the document which would have helped in easy review.I just used the .clang-format file which was available in the src folder.

**Comment by CelticMinstrel** [comment]
> Now I guess it's done properly it shows 16 deletion and 0 insertions. Sorry for the trouble I caused it is my first PR.

It's entirely possible that what you're looking at is done properly, but if that's the case, then you're not looking at the same thing I'm looking at here in this PR.

**Comment by SyedSaifuddin045** [comment]
> > Now I guess it's done properly it shows 16 deletion and 0 insertions. Sorry for the trouble I caused it is my first PR.
> 
> It's entirely possible that what you're looking at is done properly, but if that's the case, then you're not looking at the same thing I'm looking at here in this PR.

Take a look at my screenshot, It shows me just 16 deletions and 0 insertions.
I wonder what went wrong now?
Screenshot (36)

**Comment by CelticMinstrel** [comment]
Okay, now I'm looking at the same thing as you.

You deleted two blank lines in the comment at the top of the file. Could you put those back? Other than that I think it looks good now.

**Comment by SyedSaifuddin045** [comment]
> Okay, now I'm looking at the same thing as you.
> 
> You deleted two blank lines in the comment at the top of the file. Could you put those back? Other than that I think it looks good now.

My bad,fixed it now please take a look.

**Comment by stevecotton** [comment]
Looks good. Squash-merge once @soliton- reviews it?

File: src/gettext.cpp [hunk 1]
```diff
@@ -469,7 +464,6 @@ std::string dsngettext (const char * domainname, const char *singular, const cha
 	std::string msgval;
 	//Braces so that only the line that needs to be in the lock is in it:
 	{
-		std::scoped_lock lock(get_mutex());
 		msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
 	}
```
Comment (soliton-) [inline src/gettext.cpp:468]: This code should go back to `std::string msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());`

Comment (SyedSaifuddin045) [reply src/gettext.cpp]: Done Sir.🫡

File: src/gettext.cpp [hunk 2]
```diff
@@ -466,12 +461,8 @@ inline const char* is_unlocalized_string2(const std::string& str, const char* si
...
 std::string dsngettext (const char * domainname, const char *singular, const char *plural, int n)
 {
-	std::string msgval;
-	//Braces so that only the line that needs to be in the lock is in it:
-	{
-		std::scoped_lock lock(get_mutex());
-		msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
-	}
+	std::string msgval = bl::dngettext(domainname, singular, plural, n, get_manager().get_locale());
+
```
Comment (Pentarctagon) [inline src/gettext.cpp:465]: One of the things that the CI checks is that there are no lines containing only whitespace or has trailing whitespace, so you need to delete the tab here.

**Comment by soliton-** [comment]
Thanks for your contribution!

**Comment by SyedSaifuddin045** [comment]
Thanks to all of you for providing help and guiding me through this.I will never forget my first PR.😄

**Pull request merged**